spec/ajv.spec.js

```diff
@@ -377,26 +377,70 @@ describe('Ajv', function () {
 
 
   describe('addFormat method', function() {
-    it('should add format as regular expression', function() {
-      ajv.addFormat('identifier', /^[a-z_$][a-z0-9_$]*$/i);
-      testFormat();
-    });
+    describe.skip('without any type restrictions specified', function() {
```

Comment:
I don't understand. If you mean that "if type is not specified", then it is not correct. Format should be applied to all types after this change, depending on format keyword value.